src/GitVersion.Core/Configuration/BranchConfigurationCalculator.cs

```diff
@@ -89,7 +89,7 @@ private BranchConfig InheritBranchConfiguration(int recursions, IBranch targetBr
             {
                 excludedInheritBranches.Add(excludedBranch);
             }
-            var branchesToEvaluate = this.repositoryStore.ExcludingBranches(excludedInheritBranches)
+            var branchesToEvaluate = this.repositoryStore.ExcludingBranches(excludedInheritBranches, excludeRemotes: true)
                 .Distinct(new LocalRemoteBranchEqualityComparer())
                 .ToList();
```

Comment:
Please add a newline before `.ExcludingBranches` for improved legibility and consistency.

```suggestion
            var branchesToEvaluate = this.repositoryStore
                .ExcludingBranches(excludedInheritBranches, excludeRemotes: true)
                .Distinct(new LocalRemoteBranchEqualityComparer())
                .ToList();
```